docs/getting-started/configure-app-with-plugins.md

```diff
@@ -3,4 +3,32 @@ id: configure-app-with-plugins
 title: Configuring App with plugins
 ---
 
+## Adding existing plugins to your app
+
 Coming soon!
+
+### Adding a plugin page to the Sidebar
+
+In a standard Backstage app created with
+[@backstage/create-app](./create-an-app.md), the sidebar is managed inside
+`packages/app/src/sidebar.tsx`. The file exports the entire `Sidebar` element of
+your app, which you can extend with additional entries by adding new
+`SidebarItem` elements.
+
+For example, if you install the `api-docs` plugin, a matching `SidebarItem`
+could be something like this:
+
+```tsx
+// Import icon from MUI
+import ExtensionIcon from '@material-ui/icons/Extension';
+
+// ... inside the AppSidebar component
+<SidebarItem icon={ExtensionIcon} to="api-docs" text="APIs" />;
+```
+
+You can also use your own SVGs directly as icon components. Just make sure they
+are sized correctly, and set the extension to `.icon.svg`, for example:
```

Comment:
what is correctly? is it the 48x48 size or whatever the standard was?

Reply:
Updated with actual size and pointer to docs :p